src/file.cpp

```diff
@@ -49,6 +50,15 @@ inline static std::string wstring_to_string(const std::wstring &wstr)
 # endif
 #endif
 
+#ifdef _WASM
+inline static std::string get_cwd_from_env()
+{
+  char* value = getenv("PWD");
```

Comment:
it's common to have `.` as a preopen directory, you might want to check for it first.

Reply:
libsass logic changes/normalize paths -- it might be an issue to preserve .

Reply:
its ok for it to be normalized to a full path. think it would be reasonable to try using `realpath(".")` as the first check.

Reply:
Added option to avoid removal "./" for the `_WASM` in the `make_canonical_path()` -- "./" shall work now.